<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureProperties</class>
 <widget class="QWidget" name="TextureProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>287</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelFile">
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="widgetFile" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ReferenceComboBox" name="file"/>
        </item>
        <item>
         <widget class="QToolButton" name="fileNew">
          <property name="toolTip">
           <string>Save As</string>
          </property>
          <property name="icon">
           <iconset theme="document-save-as"/>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="fileBrowse">
          <property name="toolTip">
           <string>Open</string>
          </property>
          <property name="icon">
           <iconset theme="document-open"/>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTarget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="DataComboBox" name="target"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelFormat">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="DataComboBox" name="formatType"/>
     </item>
     <item row="3" column="1">
      <widget class="DataComboBox" name="formatData"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelWidth">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ExpressionLineEdit" name="width"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelHeight">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="ExpressionLineEdit" name="height"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelDepth">
       <property name="text">
        <string>Depth</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="ExpressionLineEdit" name="depth"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelLayers">
       <property name="text">
        <string>Layers</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="ExpressionLineEdit" name="layers"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelSamples">
       <property name="text">
        <string>Samples</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="samples">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelFlipVertically">
       <property name="text">
        <string>Flip Vert.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>flipVertically</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="flipVertically">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>on load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataComboBox</class>
   <extends>QComboBox</extends>
   <header>session/DataComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ReferenceComboBox</class>
   <extends>QComboBox</extends>
   <header>session/ReferenceComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ExpressionLineEdit</class>
   <extends>QLineEdit</extends>
   <header>session/ExpressionLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
